artist_query/lib/artist_query.ex

```diff
@@ -0,0 +1,41 @@
+defmodule ArtistQuery do
+  def main(_args) do
+    artist = prompt
+    load_songs("music.csv")
```

Comment:
Up to you. I didn't say that was a necessary feature, but I think a constant is better than the filename being hardcoded into some function. At least with a constant it's easier to reuse so that if it's changed later there's only one place you need to change it.